### PR TITLE
Bulk print updates

### PR DESCRIPTION
**Pull request by ikenna-hmcts**
https://tools.hmcts.net/jira/browse/PRO-4297

- Updated grantCover template

- Updated addAllUserRoles to contain system user

- Update docker-compose config for send-letter service and database

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: docker/docker-compose-with-ccd.yml [hunk 1]
```diff
@@ -3,61 +3,65 @@ networks:
...
     links:
     - send-letter-database
     depends_on:
-    - send-letter-database
+      send-letter-database:
+        condition: service_healthy
 
   send-letter-database:
-    image: docker.artifactory.reform.hmcts.net/reform/send-letter-database
+    image: postgres:9.6-alpine
```
Comment (reddy1praveen) [inline docker/docker-compose-with-ccd.yml]: try to use same `shared-database:` if possible.
I think you can create table in it.

**Review by reddy1praveen — CHANGES_REQUESTED** [review]

File: docker/docker-compose-with-ccd.yml (hunk 1 above)
Comment (ikenna-hmcts) [reply docker/docker-compose-with-ccd.yml]: If you mean the init0-db.sh then no. Could not find a way to make it work as the bulk print db needs specific configs ( ROLE, PASSWORD, DATABASE) are all different.

Comment (ikenna-hmcts) [reply docker/docker-compose-with-ccd.yml]: Will try to create the database from there.

Comment (ikenna-hmcts) [reply docker/docker-compose-with-ccd.yml]: Also can't create table from the script as the db needs to exist

Comment (malinovsky) [reply docker/docker-compose-with-ccd.yml]: I agree with Praveen, we should reuse `shared-database` container. It already has 4 CCD and 1 IDAM database, I can't see why we can't have `send-letter` database there.

Comment (ikenna-hmcts) [reply docker/docker-compose-with-ccd.yml]: It's now using the shared-database. Thanks

**Review by malinovsky — APPROVED** [review]

**Pull request merged**